src/loconet/loconet_rx.c

```diff
@@ -10,13 +10,13 @@
  */
 
 #include "loconet_rx.h"
+#include "loconet_messages.h"
```

Comment:
Done. I also updated the other includes, so that everything is following the layer pattern:
- loconet. h only includes loconet_hw.h
- loconet_rx.h only includes loconet.h
- loconet_tx.h only includes loconet.h